### ISSUE TITLE
Application Icon

### ISSUE THREAD

**Issue by afritz1** [afritz1/OpenTESArena#66]
For the program's icon, I've just been using the one that comes with the CD version, but I think it'd be neat to have a new custom-made icon instead. If anyone's a particularly good artist, maybe we could have a couple different designs done and then we'd choose the best one. I imagine most artists don't have a GitHub account, so I'll ask around.

The final icon would be submitted somewhere (like Dropbox?) in .png and .ico format. The dimensions should be something like 48x48 or 64x64.

Here's the current one for reference. It's from the CD version, just a clip of the front cover. The new one could be something completely different if we wanted.
![Icon](https://raw.githubusercontent.com/afritz1/OpenTESArena/e897c8e2fb2a2c7f3bc1f66ab68a6f20e061264f/icon.ico)

Some initial thoughts I had (just to ballpark it):
- A remake of the mage on the [front cover](http://en.uesp.net/wiki/File:ArenaDeluxe.jpg)
- A remake of the helmet on the [world map](http://en.uesp.net/wiki/File:AR-map-small.png)
- A remake of something iconic from one of the main quest dungeon splashes (like [Fang Lair](http://en.uesp.net/wiki/File:AR-quest-FangLair.jpg) or the [Crypt of Hearts](http://en.uesp.net/wiki/File:AR-quest-CryptOfHearts.jpg))
- Maybe transparency in the background

**Comment by Translator5** [comment]
Whats about "[TES: Arena - Dock Icon 1](http://blakegedye.deviantart.com/art/TES-Arena-Dock-Icon-1-255854271)" by [blakegedye](http://blakegedye.deviantart.com/)
![tes__arena___dock_icon_1_by_blakegedye-d48buf3](https://user-images.githubusercontent.com/9306861/26942850-dba426be-4c83-11e7-9967-38e1709966a7.png)
Copyright note;
"**By all mean download this icon, do what you will with it n' stuff... but credit to me would be wicked :)**"

Or sth. like "[The Elder Scrolls I - Arena](http://wordsmithmkuk.deviantart.com/art/The-Elder-Scrolls-I-Arena-279245748)" by [WordsmithMKUK](http://wordsmithmkuk.deviantart.com/)
![the_elder_scrolls_i___arena_by_wordsmithmkuk-d4m97ec](https://user-images.githubusercontent.com/9306861/26943161-ec59ef88-4c84-11e7-90e9-aa755ba63f61.png)
Copyright note;
"**unknown**"

Personally I prefer the arena icon of "[The Elder Scrolls series icons](http://kingreverant.deviantart.com/art/The-Elder-Scrolls-series-icons-161685128)" by [KingReverant](http://kingreverant.deviantart.com/)
![tes 1 arena](https://user-images.githubusercontent.com/9306861/26945961-e277362a-4c8d-11e7-98d4-0fe714d38870.png)
Copyright note;
"**Some rights reserved. This work is licensed under a
Creative Commons Attribution-Noncommercial-No Derivative Works 3.0 License.**"

Further thoughts by me:
- Sth. of this cover [here](http://media.moddb.com/images/games/1/18/17683/Arenaboitefr.jpg)
- Minimalistic-styled icon using the helmet or the map

**Comment by afritz1** [comment]
Those are nice variations on the original box art. I think I would prefer something along the lines of an image that isn't too busy. That is, it would probably have just one character or object that's the focus. It would be hard to see the crowd of characters in a 64x64 icon.

Now that I think about it, I'm leaning more towards the dragon symbol in the [Fang Lair](http://en.uesp.net/wiki/File:AR-quest-FangLair.jpg) splash screen that I mentioned earlier (also used in Morrowind and Skyrim), maybe with a different color palette, or transparent. Either that, or the gargoyle from the [Crypt of Hearts](http://en.uesp.net/wiki/File:AR-quest-CryptOfHearts.jpg).

**Comment by Translator5** [comment]
> Now that I think about it, I'm leaning more towards the dragon symbol in the Fang Lair splash screen that I mentioned earlier (also used in Morrowind and Skyrim), maybe with a different color palette, or transparent.

Is this going into the right direction?
![icon_v1](https://user-images.githubusercontent.com/9306861/26955081-9450f680-4cb4-11e7-9891-0b2605c49759.png)

**Comment by magnusroe** [comment]
These last two ones look very Morrowinny to me..

**Comment by afritz1** [comment]
Yes! That's a lot closer to what I was thinking. Did you make that yourself? I like the colors, and it's immediately recognizable. We could experiment with that design some and use something like it as the "official" project image.

Some ideas:
- The 64x64 version would not have any text (it'd be too small to read).
- A subtle light-to-dark gradient along the gray background.
- Make the dragon symbol a little more metallic with the same rusty texture.
- The color of "OpenArena" could contrast a bit more from the dragon so it's easier to see the "A", or maybe each letter could have a slightly more pronounced outline.

There's already another project called [OpenArena](https://github.com/OpenArena), so we probably don't want to reuse the name. I know people like to say "OpenDaggerfall" and "OpenMorrowind", but "OpenArena" is already taken, sadly. Perhaps it could say "Open The Elder Scrolls: Arena"? I don't know.

I wonder if there would be any intellectual property issues with using Bethesda's dragon. OpenMW's icon resembles Morrowind's icon with Moon and Star, so... I suppose there wouldn't be any problems. I actually don't know what Arena's original icon is. I've just seen the one from the DOSBox version. Maybe there was never one made.

You know, I think players of later Elder Scrolls games would be surprised to see that the dragon symbol also appears in Arena.

**Comment by magnusroe** [comment]
I don't think Arena had an icon because it was DOS only.
How about "Open TESA" "TES Arena" or something? Open Scrolls Arena? :P

But with the whole dragon in diamond shape thing, I think these two last logos are hard to tell apart from the Morrowind logo.

**Comment by afritz1** [comment]
You're right about the dragon looking more like it's from Morrowind than from Arena. We haven't found any symbols that distinctly represent "Arena" yet, so we've just been brainstorming from some of Arena's main quest splash screens and box art. Maybe another option would be the wyvern on the manual cover like @Translator5 suggested, but I think that one would be a lot harder to draw!

**Comment by Translator5** [comment]
@magnusroe Yes I used a Morrowind themed background. Also a slightly modified Skyrim logotype.
I will edit it today again, the claws next to the wings will be part of the wings and the scrag more straight, tail parted into three angles to be more similar to the dragon on the [Fang Lair](http://en.uesp.net/wiki/File:AR-quest-FangLair.jpg) picture.

>  How about "Open TESA" "TES Arena" or something? Open Scrolls Arena? :P

:rofl: Unrolled Elder Srolls: Arena?
Open TESA listens good to me, but really don't know about the writing on a icon. 

@afritz1 
> but I think that one would be a lot harder to draw!

Actually cut-out & paste works fine. 
But what I was thinking about is using maybe the lion on the walls in the first dungeon of the official game - or some of the religion icons. I will start TES Arena trough DOSbox today for the first time :rofl: and will try to find sth. unique and representative.

**Comment by afritz1** [comment]
I like both of those icons. The wyvern one has an interesting foreground and background, and the second one is nice and minimalistic, although it seems it would fit better in a Morrowind project than an Arena project. Maybe a different color palette would work nicer. The colors found in the world map feel like Arena's primary colors to me, so maybe use some combination of red, forest green, dark blue, brown, and tan instead of Morrowind's colors.

Having the whole sentence "Open The Elder Scrolls Arena" does seem a bit long. I think we'd also want to avoid having "The Elder Scrolls" shown in the icon, simply for the same reason OpenMW's name doesn't contain it. I am curious why the name "Daggerfall Unity" is fine, though. It also seems to be okay to use "TES" in the name, like with [TESRenewal](https://tesrenewal.com/). "TESA" sounds a little strange to me, but maybe "TES: Arena" would be better?

**Comment by afritz1** [comment]
The two icons in your edit are looking really interesting. I think we're getting pretty close to a reasonable style. For the one with the red "RG", if you make the stone background just a little darker, and the red letters a little more prominent, then it'll be about right and I can test it out. Could we also try a variation without the shadow?

You might want to edit a couple parts of the background as well so it's not just a copy-paste of the texture from the game assets.

**Comment by Translator5** [comment]
> By the way, any idea what "RG" from the manual cover means?

"Roht Geth" - maybe just randomly chosen letters :/ ... according to the [Imperial Library](https://www.imperial-library.info/content/daedric-alphabet) is is translated like RG and is pronounced the same.

> make the stone background just a little darker, and the red letters a little more prominent
![logos_icon_v9](https://user-images.githubusercontent.com/9306861/27013592-fe61cf0c-4ee6-11e7-91cf-f28405128230.png)
![logos_icon_v10](https://user-images.githubusercontent.com/9306861/27013593-02ff6902-4ee7-11e7-8e05-a2cfb9292d30.png)

> without the shadow
![logos_icon_v10_ns](https://user-images.githubusercontent.com/9306861/27013629-6c6f6216-4ee7-11e7-8b8e-caad8d4e24b2.png)

also without the text - blue and red . just save the pictures on your desk and look on the icon it looks pretty interesting (but not if you are scoping them for 100%) :laughing: 
![logos_icon_v11_ns](https://user-images.githubusercontent.com/9306861/27013608-49749ff6-4ee7-11e7-9ce6-857181b3a88a.png)
![logos_icon_v16_ns](https://user-images.githubusercontent.com/9306861/27013606-49722d0c-4ee7-11e7-9a60-460cb5f9ad1b.png)
![logos_icon_v14](https://user-images.githubusercontent.com/9306861/27013607-4972ebfc-4ee7-11e7-8390-1c0955f78b63.png)

**Comment by afritz1** [comment]
Here are my results after messing around with it in GIMP for a while. I think it turned out okay. I decided I liked the rough stone edges, so I removed the border.
![opentesarena_logo](https://user-images.githubusercontent.com/6729557/27019043-bcb35370-4ee9-11e7-964d-b6cc97fc3f7f.png)

And for the 64x64 icon:
![icon_logo](https://user-images.githubusercontent.com/6729557/27019045-c0066cd8-4ee9-11e7-82a8-52b2d97d78fe.png)

Of course we can just change the icon again whenever we want, but this seems decent enough for now! Thanks for helping with the process, @Translator5.

**Comment by noabody** [comment]
I really like that one but it looks busy on my light color scheme so I just borrowed the [daedric font](https://www.dafont.com/daedra.font) and did a little two tone in Pinta on Linux.  [unibuild openarena](https://raw.githubusercontent.com/noabody/unibuild/master/data/openarena.svg).

Off-topic:
I build my own package for Manjaro Linux with a custom launch script that accepts the game data path in my ```basedir``` file then pushes the information into ```$HOME/.config/OpenTESArena/options/options-changes.txt```.  I actually found it easier to copy all of ```options-defaults.txt``` into ```options-changes.txt```.  

Hackish.  Probably should have used command-line options to specifiy the path to the game data.  Lazy.
